lib/index.js

```diff
@@ -223,6 +223,9 @@ class Hatch {
             for (let outgoing_asset_id of asset.get_dependent_assets())
                 outgoing.add(outgoing_asset_id);
         });
+        this._videos.forEach((video) => {
```

Comment:
Isn't a video just another "dependent asset" like the list returned by `get_dependent_assets`? In other words, why treat this type differently from the rest?